manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -151,7 +152,9 @@ windows:
       maxUnavailable: 1
 
   kubeletRootDir: C:\var\lib\kubelet
-  providersDir: C:\k\secrets-store-csi-providers
+  providersDir: C:\\k\\secrets-store-csi-providers
+  additionalProvidersDirs:
+    - C:\\k\\secrets-store-csi-providers-alt
```

Comment:
Due to `mountPath: "{{ $path }}"` in the pod `volumeMounts`, we cannot have the same path twice.

I can either leave `additionalProvidersDirs` empty on windows or maybe theres a better path for this that we should be moving to as well?

Reply:
+1 for leaving it empty in case of windows